### PR TITLE
Update to header bar

### PR DESCRIPTION
**Pull request by hannako**
- trello card: https://trello.com/c/uDMK8vZY/194-add-datagovuk-to-black-header-bar-s
- add vertical separator between old and new service name in global header

# Before
screen shot 2018-03-06 at 13 37 41

# After 
screen shot 2018-03-06 at 13 36 34

### CODE REVIEW

File: app/views/layouts/_vertical_separator.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg id="vertical-separator" width="3" height="22">
+  <line x1="1.7" y1="0" x2="1.7" y2="22"></line>
+</svg>
```
Comment (maxf) [inline app/views/layouts/_vertical_separator.html.erb]: We don#t normally use svg, because legacy browsers. You could use a left border on an inline-div?

Comment (maxf) [reply app/views/layouts/_vertical_separator.html.erb]: We did it before, at some point, in one of the alphas. Blanca will remember.

**Comment by andrewgarner** [comment]
We need to think how this will work responsively:

screen shot 2018-03-06 at 14 24 14

**Comment by hannako** [comment]
@andrewgarner @maxf 

I can't replicate a screen size as small as you have in the screenshot. I've shrunk as far as possible in the chrome dev tools, and the only issue was the beta tag falling off the page. So now it's wrapping at screens smaller than 641.

**Comment by andrewgarner** [comment]
@hannako I checked using the Responsive Design Mode in Safari and set it to iPhone SE. You can achieve similar in Chrome by setting it to "Responsive" then choosing "Mobile S".

**Comment by hannako** [comment]
@maxf @andrewgarner 

For small screens, this is how it looks:

screen shot 2018-03-06 at 19 35 47

Happy with that?

**Comment by laurentqro** [comment]
@BlancaTortajada ☝️ looks good?

**Comment by BlancaTortajada** [comment]
@hannako @laurentqro 2 changes to be made please:
1. At the moment "Menu" dropdown is not floating on the right, just right aligned. Could you make it float so the Menu is aligned with the Beta tag?
2. The vertical spacing between the beta tag and "Find open data" should be the same as the spacing between "data.gov.uk" and "Find open data"

**Comment by hannako** [comment]
@BlancaTortajada 

Max has requested we create a new card for your style notes and merge this as is for the time being.

**Pull request merged**